webpack/scenes/Content/ContentActions.js

```diff
@@ -0,0 +1,17 @@
+import { API_OPERATIONS, get } from 'foremanReact/redux/API';
+import api from '../../services/api';
+import { CONTENT_KEY, CONTENT_TYPES_KEY } from './ContentConstants';
+
+export const getContent = contentType => get({
+  type: API_OPERATIONS.GET,
+  key: CONTENT_KEY,
+  url: api.getApiUrl(`/${contentType}`),
+});
```

Comment:
```suggestion
export const getContent = (contentType, params) => get({
  type: API_OPERATIONS.GET,
  key: CONTENT_KEY,
  params,
  url: api.getApiUrl(`/${contentType}`),
});
```

This will make pagination work..The pagination props are passed as params to the API.